package.json

```diff
@@ -155,6 +155,7 @@
     "react-router-dom": "^5.2.0",
     "react-test-renderer": "^17.0.1",
     "remark": "^13.0.0",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
This is the dependencies of the mono-repo, it's private. It's not the dependencies of the packages.

```suggestion
```